src/lib.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! # Examples
 //!
-//! ```
+//! ```ignore
```

Comment:
Instead of ignoring this example you can use the `futures` traits

```diff
diff --git a/src/lib.rs b/src/lib.rs
index 8593b93..345f539 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -7,3 +7,3 @@
 //! use async_pipe;
-//! use tokio::io::{AsyncWriteExt, AsyncReadExt};
+//! use futures::io::{AsyncWriteExt, AsyncReadExt};
 //!
```

Then the example compiles with the default flags.

Reply:
Cool, thanks!

Reply:
Interesting, the `futures` traits do not play well with `tokio` which is still used in the doc test for spawning and executing. On macOS things are fine though, but on Linux the doc test never finishes (https://github.com/routerify/async-pipe-rs/actions/runs/1093123889).

I've tried to play with cfg attrs and conditional compilation, but unsuccessfully, and honestly it's not really worth the trouble. I'm just gonna revert back to tokio and because our CI uses `cargo test --all-features` it's actually fine.